appscale/tools/parse_args.py

```diff
@@ -216,6 +216,25 @@ def add_allowed_flags(self, function):
         choices=self.ALLOWED_GCE_INSTANCE_TYPES,
         help="the Google Compute Engine instance type to use")
 
+      # Microsoft Azure specific flags
```

Comment:
Is there a validation step for these params like in def validate_infrastructure_flags() below?